WooCommerce/Classes/ViewRelated/Dashboard/Settings/Privacy/PrivacySettingsViewController.swift

```diff
@@ -41,10 +53,44 @@ class PrivacySettingsViewController: UIViewController {
         configureSections()
 
         registerTableViewCells()
+
+        loadAccountSettings()
+    }
+
+    @IBAction func pullToRefresh(sender: UIRefreshControl) {
+        loadAccountSettings {
+            sender.endRefreshing()
+        }
     }
 }
 
 
+// MARK: - Fetching Account & AccountSettings
+private extension PrivacySettingsViewController {
+
+    func loadAccountSettings(completion: (()-> Void)? = nil) {
+        guard let defaultAccount = StoresManager.shared.sessionManager.defaultAccount else {
+            return
+        }
+
+        let userID = defaultAccount.userID
+
+        let action = AccountAction.synchronizeAccountSettings(userID: userID) { [weak self] (accountSettings, error) in
+            guard let self = self,
+                let accountSettings = accountSettings else {
+                    return
+            }
+
+            // Switch is off when opting out of Tracks
+            self.collectInfo = !accountSettings.tracksOptOut
+
+            completion?()
+        }
+
+        StoresManager.shared.dispatch(action)
```

Comment:
just wondering, since `StoresManager` seems to be used quite a lot throughout the app, do we usually DI the `StoresManager` instance to a class that needs it, or access it via `StoresManager.shared` singleton?

Reply:
At the moment, it is a singleton.